setup.cfg

```diff
@@ -1,6 +1,6 @@
 [easy_install]
 zip_ok = False
 
-[pytest]
+[tool:pytest]
```

Comment:
This is the more modern way to specify tool config in more recent versions of ``setuptools``.